tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -179,3 +187,16 @@ spec:
         done
         echo -n "${RESULTS_JSON}" | tee $RESULTS_FILE
         printf 'Completed "%s" for "%s"\n\n' "$(context.task.name)" "$application"
+      volumeMounts:
```

Comment:
Are you sure this will do the trick? 

This is what we do in our [Dockerfile](https://github.com/konflux-ci/release-service-utils/blob/main/Dockerfile#L51) to enable a new cert:
```
ADD data/certs/2015-IT-Root-CA.pem data/certs/2022-IT-Root-CA.pem /etc/pki/ca-trust/source/anchors/
RUN update-ca-trust
```
We use a different subdirectory, but also run `update-ca-trust`. So I wonder if what you propose will work.

Reply:
I confirmed it to be working.

Doing it this way avoids the need to modify the task itself in order to copy things around and avoids writing into the container filesystem after the container is started.

Reply:
OK, thanks for confirming. Can you add a test for this? You can create a custom-named configMap in tasks/push-snapshot/tests/pre-apply-task-hook.sh - similar what we do e.g. for create-pyxis-image:
```sh
# Create a dummy pyxis secret (and delete it first if it exists)
kubectl delete secret test-create-pyxis-image-cert --ignore-not-found
kubectl create secret generic test-create-pyxis-image-cert --from-literal=cert=mycert --from-literal=key=mykey
```
 And then use it in a test and then in tasks/push-snapshot/tests/mocks.sh in the cosign function you can add a special case for this where if some custom pull spec is used, you will confirm that the correct thing is mounted (based on the configMap you create and use in the test).

Reply:
I'll give it a try. Thanks for the directions!

Reply:
@mmalina have a look and let me know if this is what you had in mind.

Reply:
That's exactly what I had in mind, great!